app/views/shared/webchat_message/_footer.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if TradeTariffFrontend.webchat_enabled? %>
+    <div>
+        Use our
+        <%= link_to('Digital Assistant (opens in new tab)', "#{TradeTariffFrontend.webchat_url}", target: '_blank') %>
```

Comment:
Small thing, but there's a bit of duplication with this link. Could you maybe add a WebchatHelper and consolidate there?

Reply:
there you are:

```ruby
module WebchatHelper
  def webchat_link
    link_to('Digital Assistant (opens in new tab)', "#{TradeTariffFrontend.webchat_url}", target: '_blank', rel: 'noopener')
  end

  def webchat_visible_in_footer?
    %w[commodities headings subheadings chapters sections].include?(controller_name)
  end
end
```